pywinauto/controls/HwndWrapper.py

```diff
@@ -461,11 +462,50 @@ def send_message(self, message, wparam = 0, lparam = 0):
     # Non PEP-8 alias
     SendMessage = send_message
 
+    # -----------------------------------------------------------
+    def send_chars(self,
+                   message,
+                   with_spaces=True,
+                   with_tabs=True,
+                   with_newlines=True):
+        """
+        Silently send a string to the control
+
+        Parses modifiers Shift(+), Control(^), Menu(%) and Sequences like "{TAB}", "{Enter}"
+        For more information about Sequences and Modifiers navigate to SendKeysCtypes.py
+        """
+
+        keys = SendKeysCtypes.parse_keys(message, with_spaces, with_tabs, with_newlines)
+
+        for key in keys:
+
+            vk, scan, flags = key.get_key_info()
+
+            lparam = scan << 16 | flags << 24
+
+            if isinstance(key, SendKeysCtypes.VirtualKeyAction):
+                # Shift, Left, Back, Delete, ...
+                if key.down:
+                    lparam = lparam | flags << 24 | 0 << 29 | 0 << 31
+                    win32api.SendMessage(self.handle, win32con.WM_KEYDOWN, vk, lparam)
```

Comment:
`WM_KEYDOWN` and `WM_KEYUP` should NOT work for minimized window (because it's the same that `type_keys` does). Maybe you mean `WM_CHAR` too in this case?